packages/i18n-utils/src/localize-url.tsx

```diff
@@ -60,11 +63,37 @@ const prefixLocalizedUrlPath = (
 	}
 
 	if ( validLocales.includes( localeSlug ) && localeSlug !== 'en' ) {
-		url.pathname = localeSlug + url.pathname;
+		if ( prefixOrSuffix === 'prefix' ) {
+			url.pathname = localeSlug + url.pathname;
+		} else if ( prefixOrSuffix === 'suffix' ) {
+			url.pathname = url.pathname + localeSlug;
```

Comment:
This works correctly only when `url.pathname` ends with `/` but that's not guaranteed, is it?

Reply:
I think you’re right! I’ve fixed this, and also now respect the trailing-slashedness of the path when suffixing the locale.